app/routes/auth_routes.py

```
@@ -0,0 +1,431 @@
+                'role': payload.get('role')
+            },
+            'stats': user_stats
+        })
+        
+    except Exception as e:
+        logger.error(f"Error getting user profile: {e}")
+        return jsonify({'error': 'Failed to get user profile'}), 500
+
+
+@auth_bp.route('/user/limits', methods=['GET'])
+def get_user_limits():
```

Pylint (3.3.7)

[refactor] Too many local variables (22/15)

(R0914)